frontend/public/components/persistent-volume.jsx

```diff
@@ -1,23 +1,47 @@
 import * as React from 'react';
+import { Link } from 'react-router-dom';
 
 import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
-import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLink, ResourceSummary, Timestamp } from './utils';
+import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLink, ResourceSummary, Timestamp, StatusIcon } from './utils';
+
+const pvPhase = pv => pv.status.phase;
 
 const { common } = Kebab.factory;
 const menuActions = [...common];
 
+const PVStatus = ({pv}) => {
+  const phase = pvPhase(pv);
+  return <StatusIcon status={phase} />;
+};
+
 const Header = props => <ListHeader>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.labels">Labels</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-3" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-3" sortField="status.phase">Status</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-3" sortField="spec.capacity.storage">Capacity</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-3" sortField="metadata.labels">Labels</ColHead>
   <ColHead {...props} className="col-sm-4 hidden-xs" sortField="metadata.creationTimestamp">Created</ColHead>
 </ListHeader>;
 
 const kind = 'PersistentVolume';
 const Row = ({obj}) => <div className="row co-resource-list__item">
-  <div className="col-sm-4 col-xs-6">
+  <div className="col-sm-2 col-xs-3">
     <ResourceLink kind={kind} name={obj.metadata.name} namespace={obj.metadata.namespace} title={obj.metadata.name} />
   </div>
-  <div className="col-sm-4 col-xs-6">
+  <div className="col-sm-2 col-xs-3">
+    { obj.spec.claimRef ?
```

Comment:
I'd use `_.isEmpty` so that we wouldn't try to show the claim for `claimRef: {}`. (Not sure if the API would ever return that, but there's no disadvantage to using `_.isEmpty`.)

```suggestion
    { _.isEmpty(obj.spec.claimRef) ?
```

You would need to flip the order of the items below, though.

Reply:
Let me test with code suggestion,